tests/test_dagmc_query.py

```diff
@@ -68,9 +68,11 @@ def test_get_entities_incorrect_dim():
     with warnings.catch_warnings(record=True) as w:
         warnings.simplefilter('always')
         three_vols_query = dq.DagmcQuery(three_vols, vert)
-        if len(w) == 1:
+        if len(w) == 2:
             test_pass[0] = True
-            if 'Meshset is not a volume nor a surface! Rootset will be used by default.' in str(w[-1].message):
+            if 'Meshset is not a volume nor a surface!' in str(w[0].message) and \
```

Comment:
Let's add a test to the this where a list of meshsets (maybe one volume and one surface) is passed in.

Reply:
idk why this comment is appearing here or why I put it there, but I think this test is still needed.